src/Chainweb/Pact/Types.hs

```diff
@@ -150,6 +147,12 @@ instance FromJSON MinerInfo where
   parseJSON = withObject "MinerInfo" $ \o ->
     MinerInfo <$> o .: "m" <*> (KeySet <$> o .: "ks" <*> o .: "kp")
 
+emptyPayload :: PayloadWithOutputs
```

Comment:
Is it better for discoverability that you return emptyPayload back to where? It seems like it should be there instead of here.

Reply:
I would, except that it's used in multiple modules (and hence exported from here).